src/blocks/reset.css

```diff
@@ -0,0 +1,19 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card_review {
+  margin: 0;
+}
+
+.card__price-name {
+  margin: 0;
+}
+
+.card__price-value {
+  margin: 0;
+}
```

Comment:
will be better to reset margins to tag p instead of reset them to these elements